packages/react-dom/src/__tests__/ReactDOMFiber-test.js

```diff
@@ -1132,7 +1132,16 @@ describe('ReactDOMFiber', () => {
         expect(() => {
           node.click();
         }).toErrorDev(
-          'Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.',
+          gate(flags => flags.enableEagerRootListeners)
+            ? [
+                'Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.',
```

Comment:
Not sure what this is about. I think it's because we attach more top level listeners now but why wasn't this surfaced when the flag was off during variant tests? Probably some other flag interfering.